docs/installation.rst

```diff
@@ -43,7 +22,7 @@ You can install the latest stable release build of perses via the ``conda`` pack
 
 .. code-block:: none
 
-   $ conda config --add channels omnia
+   $ conda config --add channels conda-forge
```

Comment:
We also need to mention the openeye-toolkits installation step if we don't already!
```bash
conda install -c openeye openeye-toolkits
```
Or, if we want folks to be able to `conda update` everything, including the toolkit, we should add `openeye` to the `conda config --add channels` and then add `conda install --yes openeye-toolkits` somewhere.

Reply:
good catch! I added `openeye` to the list of channels to add